dist-config/src/main/java/org/keycloak/config/AllOptions.java

```diff
@@ -0,0 +1,13 @@
+package org.keycloak.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AllOptions {
+
+    public final static List<Option<?>> ALL_OPTIONS = new ArrayList<>();
+
+    static {
+        ALL_OPTIONS.addAll(HttpOptions.ALL_OPTIONS);
```

Comment:
Correct, and it can/will be used by the following modules.

I aim to get this first "example" in, and to bulk move the encoding of all the options in a follow-up PR to help with reviews and split up work.